test/global_test.go

```diff
@@ -21,6 +22,10 @@ func init() {
 	logger.Default = logger.Discard
 }
 
+const (
+	STORAGE_BACKEND_ENVVAR = "STORAGE_BACKEND"
```

Comment:
go doesn't use `ALL_CAPS` to identify constants.  This should be StorageBackendEnvVar or (`storageBackendEnvVar` because it doesn't need to be accessible outside the package).